tests/requirements.txt

```diff
@@ -23,6 +23,7 @@ pyramid>=2.0
 pytest>=6.2.4
 pytest-celery
 redis>=3.5.3
+sanic>=19.0.0
```

Comment:
The actual instrumentation is tested from the 19 up, because the customer uses 19.03.01. A couple of if blocks could be cleaned up if we were going for 20.12, but I suppose we need customer feedback